addons/mail/static/src/components/chat_window_manager/chat_window_manager.xml

```diff
@@ -5,7 +5,7 @@
         <div class="o_ChatWindowManager" t-attf-class="{{ className }}" t-ref="root">
             <t t-if="messaging and messaging.chatWindowManager">
                 <!-- Note: DOM elements are ordered from left to right -->
-                <t t-if="messaging.chatWindowManager.hasHiddenChatWindows">
+                <t t-if="messaging.chatWindowManager.hasHiddenChatWindows and messaging.chatWindowManager.visual.hidden.isVisible">
```

Comment:
Remove field `hasHiddenChatWindows`?

For later: we need to use models (or probably chat window itself) to kill this visual thing.

Reply:
Done